### PR TITLE
Always show backtrace on Fuchsia

### PR DESCRIPTION
**Pull request by cramertj**
r? @alexcrichton 

cc @jakeehrlich

### CODE REVIEW

File: src/libstd/sys_common/backtrace.rs [hunk 1]
```diff
@@ -117,6 +117,12 @@ where
 // For now logging is turned off by default, and this function checks to see
 // whether the magical environment variable is present to see if it's turned on.
 pub fn log_enabled() -> Option<PrintFmt> {
+    // Setting environment variables for Fuchsia components isn't a standard
+    // or easily supported workflow. For now, always display backtraces.
+    if cfg!(target_os = "fuchsia") {
+        return Some(PrintFmt::Full),
```
Comment (ojeda) [inline src/libstd/sys_common/backtrace.rs]: Typo on `,` :)

Comment (cramertj) [reply src/libstd/sys_common/backtrace.rs]: Fixed, thanks.

**Pull request merged**